### ISSUE TITLE
Redirect loops and other redirect errors when retrieving titles list

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fsrrt/wikiteam#102]
```
A number of non-API downloads terminate like this:

urllib2.HTTPError: HTTP Error 302: The HTTP server returned a redirect error 
that would lead to an infinite loop.
The last 30x error message was:
Found
tail: cannot open 
`mikroitetdk_fictionpedia-20140215-wikidump/mikroitetdk_fictionpedia-20140215-hi
story.xml' for reading: No such file or directory

With variations in the 30x error code and in the error message.
One example is http://www.ifnipedia.es/index.php?title=Especial:Todas (1.16), 
another http://mikroitet.dk/wiki/Special:AllPages (also 1.16). I can't 
reproduce manually because I don't even see "Next" links, maybe that's the 
actual problem?

I've not made an audit of all such wikis because launcher.py just goes on to 
the next wiki and doesn't log the error (maybe worth another issue?), but as 
first thing better check the logic of Special:AllPages screenscraping.
```

Original issue reported on code.google.com by `nemow...@gmail.com` on 16 Feb 2014 at 8:44
- Blocked on: #104

**Comment by GoogleCodeExporter** [comment]
Original comment by `nemow...@gmail.com` on 27 Feb 2014 at 12:11
- Now blocked on: #104
